Library/Homebrew/cask/artifact/symlinked.rb

```diff
@@ -44,10 +44,11 @@ def link(**options)
                 "source '#{source}' is not there."
         end
 
-        if target.exist? && !target.symlink?
+        if target.exist?
           raise CaskError,
-                "It seems there is already #{self.class.english_article} " \
-                "#{self.class.english_name} at '#{target}'; not linking."
+                "It seems there already exists #{self.class.english_article} " \
+                "#{self.class.english_name} at '#{target}'; not overwriting. " \
+                "To install the Cask use the #{Formatter.identifier("--no-binaries")} option."
```

Comment:
Still sounds like a recommendation, as if it were a circumventable bug. It should be clear it’s a tradeoff users should make a decision on. Something like “To skip the installation of the CLI tools, use the `--no-binaries` flag”.

Reply:
Thanks, I have made your suggested change but left the wording as `option` instead of `flag` as that is what they are mostly referred to in `man brew`.